### PR TITLE
[IMP] spreadsheet_dashboard: make the sidepanel foldable

### PR DESCRIPTION
**Pull request by rrahir**
Task: 3756414

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class SpreadsheetDashboardAction extends Component {
         });
```
Comment (hokolomopo) [inline addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js:78]: probably not your fault, but vertical text creates one of the ugliset font I ever saw 
image

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ export class SpreadsheetDashboardAction extends Component {
...
         useSpreadsheetPrint(() => this.state.activeDashboard?.model);
```
Comment (hokolomopo) [inline addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js:79]: I've got a double scrollbar :x

image

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml [hunk 3]
```diff
@@ -62,4 +49,40 @@
             </t>
         </div>
     </div>
+
+    <div t-name="spreadsheet_dashboard.DashboardAction.Collapsed">
+        <div class="bg-view h-100 o_search_panel_sidebar cursor-pointer" t-on-click="toggleSidebar">
+            <div class="d-flex">
+                <button class="btn btn-light btn-sm m-1 mb-2 p-2">
+                    <i class="fa fa-fw fa-angle-double-right"/>
```
Comment (hokolomopo) [inline addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml:57]:
```suggestion
                <button class="btn btn-sm m-1 mb-2 p-2">
                    <i class="fa fa-angle-double-right"/>
```
these classes don't seems to do anything

**Review by hokolomopo — COMMENTED** [review]
Yay prettier dashboards 🥳

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml (hunk 3 above)
Comment (rrahir) [reply addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml:57]: fa-fw is indeed useless but btn-light actually affects the button backgorund when hovered ;-)

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js (hunk 2 above)
Comment (rrahir) [reply addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js:79]: somehow fixed but I don't undertand how yet ><

**Review by LucasLefevre — APPROVED** [review]
robodoo r+

**Pull request closed without merging**